TwoFHey/OTPParser/OTPParser.swift

```diff
@@ -10,10 +10,21 @@ public struct ParsedOTP {
     public let service: String?
     public let code: String
     
-    
-    func copyToClipboard() {
+    func copyToClipboard(completion: @escaping (_ some: Bool) -> Void = { _ in })  {
```

Comment:
Optional completion block for if the caller needs to do some action after the restore clipboard happens